### ISSUE TITLE
Incompatible with Webpack 5

### ISSUE THREAD

**Issue by spaceribs** [UX-and-I/storybook-design-token#50]
When adding this addon to a storybook using webpack 5, the following error occurs:

```
ERROR in ./node_modules/storybook-design-token/node_modules/postcss/lib/previous-map.js 8:35-50
Module not found: Error: Can't resolve 'path' in '/node_modules/storybook-design-token/node_modules/postcss/lib'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "path": require.resolve("path-browserify") }'
        - install 'path-browserify'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "path": false }
 @ ./node_modules/storybook-design-token/node_modules/postcss/lib/input.js 10:42-67
 @ ./node_modules/storybook-design-token/node_modules/postcss/lib/parse.js 8:36-54
 @ ./node_modules/storybook-design-token/node_modules/postcss/lib/postcss.js 18:36-54
 @ ./node_modules/storybook-design-token/dist/parsers/postcss.parser.js 54:32-50
 @ ./node_modules/storybook-design-token/dist/hooks/useTokenTabs.js 11:23-59
 @ ./node_modules/storybook-design-token/dist/components/Panel.js 27:21-53
 @ ./node_modules/storybook-design-token/dist/register.js 9:14-43
```

Adding `resolve.fallback` to the webpack configuration per the instructions does not seem to work for me.

**Comment by Sqrrl** [comment]
Thanks for reporting. I will have a closer look as soon as possible.

**Comment by yngvebn** [comment]
Seems the problem is that the Webpack config for the Storybook manager isn't pluggable, so there aren't any way of adding the resolve.fallback to the build...

**Comment by Ryuno-Ki** [comment]
Affects me, too.

Is there any update?

**Comment by Sqrrl** [comment]
I think I found the issue. Could you verify that `v1.2.0-beta.0` fixes it?

**Comment by Ryuno-Ki** [comment]
`npm i storybook-design-token@next`, enabling it in `.storybook/main.js` and I can see the Design Tokens again!
Thank you, @Sqrrl!

**Comment by Sqrrl** [comment]
Published as v1.2.0
